bears/general/LineCountBear.py

```diff
@@ -10,14 +10,24 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            exclude_blank_lines: bool=False):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
 
         :param max_lines_per_file: Number of lines allowed per file.
+        :param exclude_blank_lines: Enable to exclude blank lines.
         """
         file_length = len(file)
+
+        blank_line_counter = 0
+        file_length = len(file)
+        for line in file:
+            if line.split() == []:
+                blank_line_counter += 1
```

Comment:
This can be reduced to`len(list(filter(lambda x: re.match(r'\s+')), file)` . Actually am not sure if we'll have to convert to list explicitly to get the length, try if we can get length of an iterator ;)